h2o-core/src/main/java/water/api/schemas3/ParseSetupV3.java

```diff
@@ -79,6 +79,9 @@
   @API(help="Key-reference to an initialized instance of a Decryption Tool")
   public KeyV3.DecryptionToolKeyV3 decrypt_tool;
 
+  @API(help = "Names of the columns the persisted dataset has been partitioned by.", direction = API.Direction.INOUT)
+  public String[] partition_by;
```

Comment:
Not exposing the synthetic column API, the correct domain name is partition column, therefore `partiton_by`.